lib/liquid/context.rb

```diff
@@ -47,6 +48,14 @@ def add_filters(filters)
       @strainer = nil
     end
 
+    def add_global_filters(global_filters)
+      @global_filters = global_filters if global_filters.respond_to?(:call)
```

Comment:
same question as in the other PR (sorry if I missed the answer): why would it not respond to `:call`? and shouldn't we error in that case?